phpunit/class-rest-nav-menu-items-controller-test.php

```diff
@@ -271,59 +271,6 @@ public function test_create_item_invalid_term() {
 		$this->assertErrorResponse( 'rest_term_invalid_id', $response, 400 );
 	}
 
-	/**
-	 *
-	 */
-	public function test_create_item_change_position() {
-		wp_set_current_user( self::$admin_id );
-		$new_menu_id = wp_create_nav_menu( rand_str() );
-		for ( $i = 1; $i < 5; $i ++ ) {
-			$request = new WP_REST_Request( 'POST', '/__experimental/menu-items' );
-			$request->add_header( 'content-type', 'application/x-www-form-urlencoded' );
-			$params = $this->set_menu_item_data(
-				array(
-					'menus' => $new_menu_id,
-				)
-			);
-			$request->set_body_params( $params );
-			$response = rest_get_server()->dispatch( $request );
-			$this->check_create_menu_item_response( $response );
-			$data = $response->get_data();
-			$this->assertEquals( $data['menu_order'], $i );
-		}
-	}
```

Comment:
Should this test be removed? It doesn't seem to trigger validation.

Reply:
It failed.

Reply:
It failed because it resulted in the following sequence of positions: `0, 2, 3, 4, 5`. According to https://core.trac.wordpress.org/ticket/28140, menu position 0 is invalid and breaks things so it would be good to get it fixed. I proposed an adjustment in 4e97c84966 – what do you think @spacedmonkey ?

Reply:
I think it might be better change the schema to make the min 1 and default 1. 

https://github.com/WordPress/gutenberg/blob/8e4ebfae09351aa5d21fe072deb16d265bdb130e/lib/class-wp-rest-menu-items-controller.php#L958-L964

Reply:
I would also think about making the position a required field.

Reply:
Nice! Yes, let's do that instead

Reply:
Fixed.